cmd/dosa/schema.go

```diff
@@ -128,7 +127,7 @@ type SchemaCheck struct {
 
 // Execute executes a schema check command
```

Comment:
Let's update the documentation for this and the `SchemaCheck` struct. In fact, we may want to call it something different now since it's checking if the schema can be upgraded.

Reply:
Yeah, I will do the doc change.